frontend/src/utils/enums.js

```diff
@@ -52,6 +52,12 @@ export const rolesEnum = {
   UI: 'UI/UX Designer'
 }
 
+export const permissionRolesEnum = {
+  M: 'Member',
+  L: 'Lead',
+  D: 'Director'
```

Comment:
nit. i’d rather have it be `MEMBER` `LEAD` etc instead of M,L,D because it’s more clear what those enums mean when you are reading code. For instance, permissionRolesEnum.MEMBER is easier to understand than permissionRolesEnum.M. When reading the code i would need to go to the enums page to understand what `M` means